tests/unit/config/test_project.py

```diff
@@ -27,7 +30,127 @@
 )
 
 
-class TestProject(BaseConfigTest):
+class TestProjectMethods:
+    @pytest.fixture(scope="function")
+    def selector_config(self) -> SelectorConfig:
+        return SelectorConfig.selectors_from_dict(
+            data={
+                "selectors": [
+                    {
+                        "name": "my_selector",
+                        "definition": "give me cats",
+                        "default": True,
+                    }
+                ]
+            }
+        )
+
+    @pytest.fixture(scope="function")
+    def project(self, selector_config: SelectorConfig) -> Project:
```

Comment:
is there a desire to have this project fixture more broadly accessible for other unit tests? or is it sufficient as defined for this set of unit tests?

Reply:
Seconding this one here, I think we should consider moving both this and the Selector config into common util fixtures

Reply:
Yes there is a desire for the fixture to be available more broadly. We have an idea of what tests might use this as a fast follow, and then it would be part of that work to move this fixture into a more accessible location. Thus it is sufficient here to have it confined to the test class.